CHANGELOG.md

```diff
@@ -9,7 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [0.4.0](https://github.com/USGS/time-series-services/compare/time-series-services/compare/0.2.0...0.4.0)
 ### Added
```

Comment:
Unrelated to this PR, but since you are changing the file anyway. :-) 
Line #9 should have: time-series-services-0.2.0...time-series-services-0.4.0
Thanks

Reply:
ok